schemas/issue.json

```diff
@@ -93,6 +112,21 @@
 				"$ref": "#/definitions/SectionSummary"
 			}
 		},
+        "showNumber": {
+          "type": "boolean"
+        },
+        "showTitle": {
+          "type": "boolean"
+        },
+        "showVolume": {
+            "type": "boolean"
+        },
+        "showYear": {
+          "type": "boolean"
+        },
+        "styleFileName": {
+            "type": "string"
+        },
```

Comment:
There are some whitespace issues here that need formatting. We use tabs not spaces in the JSON files, I think.